decidim-proposals/spec/shared/admin_manages_proposal_answer_imports_examples.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+shared_examples "admin manages proposal answer imports" do
+  before do
+    page.find(".imports").click
+    click_link "Import answers from a file"
+  end
+
+  describe "import answers from a file" do
+    it "has start import button" do
+      expect(page).to have_content("Import")
+    end
+
+    it "returns error without a file" do
+      click_button "Import"
+      expect(page).to have_content("There's an error in this field")
```

Comment:
This field field works a bit differently at 0.26 and the error shown next to it is actually "can't be blank".

But I think we can match also the flash error here?
```suggestion
      expect(page).to have_content("There was a problem during the import")
```